### PR TITLE
Simple homepage and book details views

### PR DESCRIPTION
**Pull request by wdebsqi**
- added base HTML template
- moved `book_details` view and integrated it with the base template
- added `home` page view and template
- fixed the issue with tests trying to connect to the Postgres database during testing
- created unit tests for testing `home` and `book_details` views

### CODE REVIEW

File: read_it/tests/setup_data.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from ..models import Book
+
+
+def create_new_book(id: int | None = None) -> Book:
+    return Book.objects.create(
+        id=id,
+        title=f"This is a test book number {id}",
+        description="This is a test description number",
```
Comment (pmkoszalka) [inline read_it/tests/setup_data.py]: This is a test description or This is a test description of a book with and id number {id}. I think we do not provide here description number

File: read_it/tests/test_home_view.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        cls.client = Client()
+
+    def test_home_returns_200(self):
+        response = self.client.get(reverse("home"))
+        self.assertEqual(response.status_code, 200)
+
+    def test_home_context_contains_five_books(self):
+        response = self.client.get(reverse("home"))
+        self.assertEqual(len(response.context["last_published_books"]), 5)
```
Comment (pmkoszalka) [inline read_it/tests/test_home_view.py:22]: Is there any reason that you create 10 boks but only check for 5 on website?

**Review by pmkoszalka — DISMISSED** [review]

File: read_it/tests/test_home_view.py (hunk 2 above)
Comment (wdebsqi) [reply read_it/tests/test_home_view.py:22]: Yes, this test makes sure that the home page will render **only** five books, even if there are more available.

**Review by pmkoszalka — COMMENTED** [review]
LGTM!

**Pull request merged**